test/support/adapters/slack/helpers.ex

```diff
@@ -59,6 +59,8 @@ defmodule Cog.Adapters.Slack.Helpers do
         "https://slack.com/api/channels.history"
       "G" <> _ ->
         "https://slack.com/api/groups.history"
+      "D" <> _ ->
```

Comment:
Does this need to be accounted for when we're classifying IDs in the connector? Where does this code come into play?